### PR TITLE
Store the budget selection

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA: No currently able to be QAed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5605/
Test PASSed.

**Comment by anthonydillon** [comment]
Looks good :+1:

File: jujugui/static/gui/src/app/components/deployment-flow/budget/budget.js [hunk 1]
```diff
@@ -127,6 +141,7 @@ YUI.add('deployment-budget', function() {
               <juju.components.InsetSelect
                 disabled={disabled}
                 label="Budget"
+                onChange={this._handleBudgetChange}
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/budget/budget.js]: I'd kind of figure we'd only want to set the budget change on a 'save' but I'll leave that up to to the design

**Comment by hatched** [comment]
Thanks @huwshimi feel free to land this and disregard my comment if this is how it's to be :)

**Comment by huwshimi** [comment]
@hatched Ah yeah, I see what you mean, but there is no save for this step...

**Comment by huwshimi** [comment]
Thanks @anthonydillon @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**